packages/edit-site/src/components/global-styles/custom-css.js

```diff
@@ -44,6 +52,30 @@ function CustomCSSControl( { blockName } ) {
 			return;
 		}
 		setCustomCSS( value );
+		if ( cssError ) {
+			const [ transformed ] = transformStyles(
+				[ { css: value } ],
+				'.editor-styles-wrapper'
+			);
+			if ( transformed ) {
+				setCSSError( null );
+			}
+		}
+	}
+
+	function handleOnBlur( event ) {
+		if ( event?.target?.value ) {
+			const [ transformed ] = transformStyles(
+				[ { css: event.target.value } ],
+				'.editor-styles-wrapper'
+			);
+
+			setCSSError(
+				transformed === null
+					? __( 'Error while parsing the CSS.' )
```

Comment:
Some users might not know what parsing is. I wonder if there's a less technical phrase that can be used for the tooltip.

Reply:
Good point - I have changed it to `There is an error with your CSS structure.`  - does that sound better?

Reply:
Yeah, I think that's good ❤️